chromium_src/extensions/common/permissions/permissions_data.cc

```diff
@@ -9,28 +9,6 @@
 
 namespace extensions {
 
-const char k1PasswordId[] = "aomjjhallfgjeglblehebfpbcfeobpgk";
-const char k1PasswordXId[] = "aeblfdkhhhdcdjpifhhbdiojplfjncoa";
-const char kBitWardenId[] = "nngceckbapebfimnlniiiahkandclblb";
-const char kDashlaneId[] = "fdjamakpfbbddfjaooikfcpapjohcfmg";
-const char kEnPassId[] = "kmcfomidfpdkfieipokbalgegidffkal";
-const char kKeePassXCId[] = "oboonakemofpalcgghocfoadofidjkkk";
-const char kKeeperId[] = "bfogiafebfohielmmehodmfbbebbbpei";
-const char kLastPassId[] = "hdokiejnpimakedhajhdlcegeplioahd";
-const char kPainFreePasswordsId[] = "hplhaekjfmjfnfdllkpjpeenlbclffgh";
-const char kRoboFormId[] = "pnlccmojcmeohlpggmfnbbiapkmbliob";
-const char kSafeInCloudId[] = "lchdigjbcmdgcfeijpfkpadacbijihjl";
-
-bool IsKnownPasswordManagerExtension(
-    const extensions::ExtensionId& extension_id) {
-  return (extension_id == k1PasswordId) || (extension_id == k1PasswordXId) ||
-         (extension_id == kBitWardenId) || (extension_id == kDashlaneId) ||
-         (extension_id == kEnPassId) || (extension_id == kKeePassXCId) ||
-         (extension_id == kKeeperId) || (extension_id == kLastPassId) ||
-         (extension_id == kPainFreePasswordsId) ||
-         (extension_id == kRoboFormId) || (extension_id == kSafeInCloudId);
-}
-
 bool IsBraveProtectedUrl(const GURL& url) {
```

Comment:
please remove this entire file and move `IsBraveProtectedUrl` to brave_extensions_api_client I guess since that's the only place it is used

Reply:
it can be unit tested through `BraveExtensionsAPIClient::ShouldHideBrowserNetworkRequest` which needs a test anyway

Reply:
Added in last revision.